Foundation-delegation-program/validator-profiles/The Silver Fox/thesilverfox.md

```diff
@@ -10,24 +12,22 @@ As the Aura foundation delegation receiver, The Silver Fox pledges to meet the f
     6. Must set Gas Price >= 0.001 uaura
     7. Publicly announced to the social channels on the delegation decision
     8. Commit to network contributions listed below: 
-    Category 1: Infrastructure
-     - Providing a Testnet node
     Category 2: Software engineering
      - As I am going through my journey of learning Javascript, GO and RUST are on my list to learn.  I've already provided some code contributions to other chains but they are very small.
 	 - Help support node runners
 	 - *Completed* Created Unbonding stats with charts.  https://thesilverfox.pro/#blockchains Click on Aura's Icon and wait for data to load.
-		- Working on providing more data like:
+		- **Dashboard creation is in progress**  Working on providing more data like:
 			- Top 10 total unbonds from Validators
 			- Top Unbonding Wallets
 			- Hourly & Daily total staked amounts
 				- Per validator
 				- Top wallets
 				- Validators adding self bonds
 	 - Nearly complete Upgrade Script offered as a complete package to replace Cosmosvisor.
-		- Might like some help getting a separate testnet going where I can test the script by issuing upgrade props to fully simulate the process.
+		- During a part of writing a function I decided that certain parts will be beneficial if served as a API to easily get the data I want.  So I built a API for cosmos registry.  https://github.com/cosmos/chain-registry My contributions are under Web endpoints and APIs in the readme.
     Category 3: Community growth
-     - I am still working on an extensive Documentation site. *This is taking quite a bit of time*
-	 - I retweet/quote tweet most announcements.
+     - I am still working on an extensive Documentation site.
+		- I am having to restart a lot of screenshots due to the keplr version change.
     Category 4: Business development
-     - Being avaliable for Aura when needs arise
-	 - I still have every intention of writing the math paper on the DEX formula 
\ No newline at end of file
+     - I still have every intention of writing the math paper on the DEX formula
+		- I'd like to know the status and gain some insight on where the vision for it is going.
```

Comment:
Thanks, actually we already deployed the swap contract and have a DEX https://halotrade.zone